cypress/support/util/ModalUtils.ts

```diff
@@ -6,8 +6,8 @@ export default class ModalUtils {
   private cancelModalBtn = "#modal-cancel";
   private closeModalBtn = ".pf-c-modal-box .pf-m-plain";
 
-  confirmModal() {
-    cy.get(this.confirmModalBtn).click();
+  confirmModal(force: boolean = false) {
+    cy.get(this.confirmModalBtn).click({ force: force });
```

Comment:
When you pass a value with the same name as the key you can omit the value.
```suggestion
    cy.get(this.confirmModalBtn).click({ force });
```